test/unit/test.utils.js

```diff
@@ -29,18 +29,14 @@ describe("lib/utils", function () {
   });
 
   describe("getManifest", function() {
-    it("returns manifest in cwd()", function () {
-      process.chdir(simpleAddonPath);
-      return utils.getManifest().then(function(manifest) {
+    it("returns manifest in simple-addon directory", function () {
+      return utils.getManifest({addonDir: simpleAddonPath}).then(function(manifest) {
         expect(manifest.name).to.be.equal("simple-addon");
         expect(manifest.title).to.be.equal("My Simple Addon");
       });
     });
 
     it("returns manifest from custom XPI file", function () {
-      // Change into a non-package path.
-      process.chdir(path.join(__dirname, "..", "addons"));
```

Comment:
This is a bit weird but what it did was made sure the test uses `simpleAddonXPI` rather than the add-on dir. So I think you should make this additional change :
````
return utils.getManifest({
    xpiPath: simpleAddonXPI, 
    addonDir: path.join(__dirname, "..", "addons"),  // pass in an invalid dir just to make sure it's not used
  })
    ...
````